Benchmarking per channel quantization

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29627 Benchmarking per channel quantization**

Differential Revision: [D18443929](https://our.internmc.facebook.com/intern/diff/D18443929)

Commit:
Benchmarking per channel quantization

raghuramank100 commented on benchmarks/operator_benchmark/pt/quantization_test.py:
```diff
@@ -55,6 +55,42 @@ def forward(self):
+            'scales': scales,
+            'zero_points': zero_points,
+            'dtype': dtype,
+            'axis': 0
```
Suggest adding one non-zero axis argument as the implementation could be slower when the axis is not zero

Commit:
Update on "Benchmarking per channel quantization"


Differential Revision: [D18443929](https://our.internmc.facebook.com/intern/diff/D18443929)
